HerPortal/appsettings.json

```diff
@@ -41,6 +41,10 @@
 
     "S3": {
         "BucketName": "desnz-her-portal-referrals",
-        "Region": "eu-west-2"
+        "Region": "eu-west-2",
+
+        "LocalDevOnly_AccessKey": "minioadmin",
+        "LocalDevOnly_SecretKey": "minioadmin",
+        "LocalDevOnly_ServiceURL": "http://localhost:9000"
```

Comment:
Should these go in `appsettings.Development.json`, to avoid any risk they show up in the deployed environments?

Reply:
Good point, I thought that at one point, then forgot!